controls/radchartview/getting-started/data-point.md

```diff
@@ -0,0 +1,114 @@
+
+* [Series DataPoint Types](#series-datapoint-types)
+* [Sampling and DataItem](#sampling-and-dataitem)
+* [DataPoint](#datapoint)
+
+## Series DataPoint Types
+
+The following table shows what series which data point type uses.
+
+| Cartesian Chart Data Points | Pie Chart Data Points | Polar Chart Data Points |
+| --- | --- | --- |
+| __CategoricalDataPoint__<ul><li>LineSeries</li><li>StepLineSeries</li><li>SplineSeries</li><li>BarSeries</li><li>AreaSeries</li><li>SplineAreaSeries</li><li>StepAreaSeries</li><li>PointSeries</li</ul> | __PieDataPoint__<ul><li>PieSeries</li><li>DoughnutSeries</li></ul> | __PolarDataPoint__<ul><li>PolarLineSeries</li><li>PolarAreaSeries</li><li>PolarPointSeries</li><li>RadarLineSeries</li><li>RadarAreaSeries</li><li>RadarPointSeries</li></ul> |
+| __ScatterDataPoint__<ul><li>ScatterLineSeries</li><li>ScatterSplineSeries</li><li>ScatterAreaSeries</li><li>ScatterSplineAreaSeries</li><li>ScatterPointSeries</li></ul> |
+| __RangeDataPoint__<ul><li>RangeSeries</li><li>RangeBarSeries</li></ul> |
+| __OhlcDataPoint__<ul><li>OhlcSeries</li><li>CandlestickSeries</li></ul> |
+| __BubbleDataPoint__<ul><li>BubbleSeries</li></ul> |
+| __BubbleScatterDataPoint__<ul><li>ScatterBubbleSeries</li></ul> |
+
+## DataPoint
+
+DataPoint is an abstract class used as a base by the concerete data point impelementations. The class contains information related to the element plotted in the chart. There are several classes that derive from DataPoint and they all share some common properties.
+
+* __Presenter__: A property of type __IChartElementPresenter__ that holds the a reference to the parent series where the data point is visualized.
+
+* __LayoutSlot__: A property of type __RadRect__ that holds the rectangle where the data point is visualized. You can use it to get the size and the screen coordinates of the visual element representing the point.
+
+* __DataItem__: A property of type __object__ that holds a reference to the view model of the data point. This property is valid only in a [data-bound]({%slug radchartview-populating-with-data-chartdatasource%}). In a sampling scenario the property holds an object of type [DataPointSamplingInfo](#sampling-and-dataitem) instead of the view model (the data item).
+	
+* __IsSelected__: A property of type __bool__ that indicates if the data point is selected. For more information, see the [Selection]({%slug radchartview-features-selection%}) help article.
+
+* __IsInPlotRange__: A property of type __bool__ that indicates if the data point's visual is in the plot area range. This is the actual range of the axes.
+
+* __Label__: A property of type object that contains the label associated with the data point.
+
+Each different DataPoint inheritor implement properties that defines the values used to plot it in the chart.
```

Comment:
Each different DataPoint inheritor implements properties that define the values used to plot it in the chart.